cvmfs/magic_xattr.cc

```diff
@@ -72,6 +74,13 @@ MagicXattrManager::MagicXattrManager(
 
   Register("user.authz", new AuthzMagicXattr());
   Register("user.external_url", new ExternalURLMagicXattr());
+
+  Register("user.cleanup_unused_first", new CleanupUnusedFirstMagicXattr());
+  Register("user.list_open_hashes", new ListOpenHashesMagicXattr());
+  Register("user.list_groups_open_hashes",
```

Comment:
`"group" `is a slightly confusing name in this context. I would replace it with `"all_repositories"`